template-for-repository/proofs/Makefile.common

```diff
@@ -617,6 +618,11 @@ else
 endif
 
 # Optionally apply loop contracts
+#
+# When applied, the instrumentation introduces a backjump to prove both
+# the base case and then the step case on the same loop.
+# Therefore, it is necessary to unwind this twice: once for the base case,
+# and once more for the step case.
```

Comment:
Could we rephrase this to be less passive:

Unroll loops twice when applying loop contracts.  The instrumentation for loop contracts introduces a back jump to prove both the base case and the induction step of the induction, and we need to unroll both iterations.